back/engines/commercial/admin_api/app/serializers/admin_api/tenant_serializer.rb

```diff
@@ -9,17 +9,13 @@ module AdminApi
   # one DB query using:
   #   AdminApi::TenantSerializer.new(tenant, app_configuration: config)
   class TenantSerializer < ActiveModel::Serializer
-    attributes :id, :name, :host, :settings, :style, :logo, :header_bg, :favicon
+    attributes :id, :name, :host, :settings, :style, :logo, :favicon
```

Comment:
See [comment](https://github.com/CitizenLabDotCo/citizenlab/pull/2230/files#r960366946) on deleting this from `tenants`.